src/passes/OptimizeInstructions.cpp

```diff
@@ -221,6 +221,7 @@ struct OptimizeInstructions
       // `using namespace Match` can be hoisted to function scope and this extra
       // block scope can be removed.
       using namespace Match;
+      using namespace Abstract;
```

Comment:
Would using it once and for all at the top of the file be problematic?

Reply:
Probably not, but I do want to tie this change specifically to the use of matchers, since I don't think removing the explicit `Abstract::` prefix is a readability win in other cases.